client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -69,7 +69,7 @@ export function Gutenboard() {
 							</div>
 							<div>
 								<SettingsSidebar isActive={ isEditorSidebarOpened } />
-								<Sidebar.Slot />
+								<SidebarSlot />
```

Comment:
I _think_ it might be [broken](https://github.com/Automattic/wp-calypso/pull/37763#issuecomment-556989757) because of this. I think the `Slot` symbol exported from `./components/sidebar` might not be the same thing as `Sidebar.Slot`?

Reply:
Thanks, I'll check into this 👀

Reply:
The export was changed, this works fine for me and on calypso.live. Did you restart your development server?

Reply:
I didn't -- my bad :flushed: